common/lib/xmodule/xmodule/course_module.py

```diff
@@ -414,13 +414,13 @@ class CourseFields(object):
         scope=Scope.settings
     )
     has_children = True
-    info_section_name = String(
+    info_sidebar_name = String(
         display_name=_("Course Home Sidebar Name"),
         help=_(
             "Enter the heading that you want students to see above your course handouts on the Course Home page. "
             "Your course handouts appear in the right panel of the page."
         ),
-        scope=Scope.settings, default=_('Course Resources and Tools'))
+        scope=Scope.settings, default='Course Handouts')
```

Comment:
I know this is just a revert back to the old code, but should this string be i18n.  Maybe we can add a quick commit on top of this?